seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/AbstractContainer.java

```diff
@@ -64,6 +64,12 @@ public AbstractContainer() {
 
     protected abstract List<String> getExtraStartShellCommands();
 
+    protected abstract String getThirdPartyPluginsPath();
```

Comment:
What's the meaning of this? In e2e test cases, extra jars not update to github server.

Reply:
i will remove this method